### ISSUE TITLE
Batch Normalization

### ISSUE THREAD

**Issue by lfb94** [CEA-LIST/N2D2#7]
Hello, I am having a little bit of trouble understanding how to use this kind of layer and I haven't found any examples...

What I want to do is to create the following block: Conv (without activation) -> BatchNorm -> ReLU .

As I understand it, the output size after normalization should remain unchanged, so I'm not sure about the NbOutputs option. It would help to see an example of an .ini file using this kind of combination. 

Thank you!

**Comment by olivierbichler-cea** [comment]
Here is an example of what you are trying to achieve:
```
[conv1]
Input=...
Type=Conv
KernelWidth=3
KernelHeight=3
NbChannels=48
ActivationFunction=Linear  ; Linear means no activation

[batchnorm1]
Input=conv1
Type=BatchNorm
NbOutputs=48  ; same as NbChannels in conv1
ActivationFunction=Rectifier  ; Rectifier is ReLU
```
Let me now if this helps!

**Comment by lfb94** [comment]
Thank you!
